itemspawner/sh_plugin.lua

```diff
@@ -0,0 +1,69 @@
+local PLUGIN = PLUGIN
+
+PLUGIN.name = "Item Spawner System"
+PLUGIN.author = "Gary Tate"
+PLUGIN.description = "Allows staff to select item spawn points with great configuration."
+
+PLUGIN.spawner = PLUGIN.spawner or {}
+PLUGIN.items = PLUGIN.items or {}
+PLUGIN.spawner.positions = PLUGIN.spawner.positions or {}
+
+PLUGIN.items.common = {
+	"pistol"
+}
+
+PLUGIN.items.rare = {
+	"shotgun"
+}
+
+function PLUGIN:AddSpawner(client, position, title)
+	if !(client:IsSuperAdmin()) then return end
```

Comment:
You should do a privilege check via your commands using [ix.command.HasAccess](https://docs.gethelix.co/libraries/ix.command.html#ix.command.HasAccess), or [CAMI.PlayerHasAccess](https://github.com/NebulousCloud/helix/blob/543a09cc9871097a7a4c50cafc9989b84654140e/gamemode/core/sh_config.lua#L204) if you're combining your commands' access with a single privilege.